src/containers/App.tsx

```diff
@@ -75,6 +77,7 @@ class App extends React.Component {
                         {process.env.REACT_APP_CHAIN_ID === '97' && (
                           <Route exact path="/faucet" component={Faucet} />
                         )}
+                        {isDevEnvironment && <Route exact path="/dev" component={Dev} />}
```

Comment:
What is this page for?

Reply:
just for develop debugging

Reply:
Let's remove this in favor of using storybooks. Storybooks are perfect for this purpose

Reply:
removed